ngraph/frontend/paddlepaddle/src/op/split.cpp

```diff
@@ -14,25 +14,35 @@
 // limitations under the License.
 //*****************************************************************************
 
-#include <ngraph/opsets/opset6.hpp>
-#include "split.h"
+#include <ngraph/opsets/opset7.hpp>
+#include "split.hpp"
 #include <paddlepaddle_frontend/utility.hpp>
 
 namespace ngraph {
 namespace frontend {
 namespace pdpd {
 namespace op {
-    OutputVector split(const NodeContext& node) {
+    NamedOutputs split(const NodeContext& node) {
         using namespace ngraph;
-        using namespace opset6;
+        using namespace opset7;
         const auto& data = node.get_ng_input("X");
         auto dim = node.get_attribute<int32_t>("axis");
         // todo: 'num' can be list of values, in this case we should create VariadicSplit
         // todo: support VariadicSplit
         auto num_or_sections = node.get_attribute<int32_t>("num");
         auto axis = std::make_shared<Constant>(ngraph::element::i32, Shape{}, dim);
 
-        return std::make_shared<ngraph::opset6::Split>(data, axis, num_or_sections)->outputs();
+        NamedOutputs named_outputs;
+        const auto& split_outputs = std::make_shared<Split>(data, axis, num_or_sections)->outputs();
+        const auto& out_names = node.get_output_names();
```

Comment:
and same here, maybe using just 'auto' will be more understandable

Reply:
fixed